File: src/projecteuler/Problem02.java
package projecteuler;

public class Problem02 {

	public static void main(String[] args) {
		int one = 1;
		int two = 1;
		int temp = 0;
		int sum = 0;
		while (two < 4000000){
			temp = one + two;
			one = two;
			two = temp;
			if (two % 2 == 0)
				sum += two;
		}
		System.out.print(sum);
	}

}

File: Python/Problem02.py
one = 1
two = 1
thr = 0
a = 0
while ((one <= 4000000) and (two <= 4000000)):
    thr = one
    one = two + thr
    two = thr
    if (two % 2 == 0):
        a += two
print a
